### DISCUSSION TITLE
Plotman is crashing / unresponsive when destination drive gets full

### DISCUSSION THREAD

**Discussion by vvavepacket** [ericaltendorf/plotman#596] [Q&A]
I am running Plotman 24/7 and when one of my destination/remote drives (no archive) gets full, plotman just crashes and unable to interact.

Do you guys face the same issue or how do you guys handle if a destination drive is about to get full? Do you let it go full and plotman will continously attempt to write and write until it crashes? Or there is something wrong with my setup? Is it a bug?

Also, when a drive is about to get full, how do we tell plotman to stop creating new plots?

**Comment by altendky** [comment]
Roughly everyone is intended to use archiving.  We are working to make it more easily and flexibly setup.  If you are getting a crash, it would be good to share the output.  Generally plotman keeps running and the plotting processes themselves sit around waiting for space to free up.  Also, plotman doesn't really have a 'we are done plotting' feature.  If you are done plotting, just stop plotman.  Yeah, if you really really are done then you will have to cleanup a few tmp files and kill a few processes.  This could certainly be improved, but there are other things needed for _creating_ plots that seem higher priority.  `:]`
